### PR TITLE
Add .tarifa.json file and minimal settings in default ios configuration

### PR DESCRIPTION
**Pull request by peutetre**

### CODE REVIEW

File: tarifa.json [hunk 1]
```diff
@@ -12,7 +12,11 @@
       "default": {
         "id": "org.lichess.mobileapp.dev",
         "product_name": "lichess dev",
-        "product_file_name": "lichess"
+        "product_file_name": "lichess",
+        "API_END_POINT": "http://en.lichess.org",
+        "SOCKET_END_POINT": "socket.en.lichess.org",
+        "GA_ID": "xxxxxxxxxxx",
```
Comment (veloce) [inline tarifa.json]: I use `@ifdef` in preprocess to skip GA_ID on purpose, so this should be omitted in default conf.

File: tarifa.json [hunk 2]
```diff
@@ -12,7 +12,11 @@
...
+        "version": "1.0.2"
```
Comment (veloce) [inline tarifa.json]: Same for version: we don't need it in dev, so let's omit it

**Comment by peutetre** [comment]
change done

**Comment by veloce** [comment]
thanks!

**Pull request merged**